docs/self-hosting/configuration/domain-configuration.mdx

```diff
@@ -4,13 +4,13 @@ description: "Configuring your domain for Formbricks."
 icon: "globe"
 ---
 
-Formbricks supports both single domain and second domain configurations. This guide will help you set up your domains correctly.
+Formbricks supports both single domain and dual domain configurations with our new Public/Private Domain Split feature. This guide will help you set up your domains correctly for enhanced security and organization.
```

Comment:
The docs should be timeless, so please avoid terms like "new feature" or "now supports".